lms/envs/common.py

```diff
@@ -340,9 +340,6 @@
     # Enable OpenBadge support. See the BADGR_* settings later in this file.
     'ENABLE_OPENBADGES': False,
 
-    # The block types to disable need to be specified in "x block disable config" in django admin.
-    'ENABLE_DISABLING_XBLOCK_TYPES': True,
```

Comment:
Should there be an initial data migration that reads this value from config, so that during an upgrade, the behavior of the site doesn't shift until someone does something to the database?